kotlinpoet/src/main/java/com/squareup/kotlinpoet/TypeSpec.kt

```diff
@@ -462,7 +464,8 @@ public class TypeSpec private constructor(
     internal val isEnum get() = kind == Kind.CLASS && ENUM in modifiers
     internal val isAnnotation get() = kind == Kind.CLASS && ANNOTATION in modifiers
     internal val isCompanion get() = kind == Kind.OBJECT && COMPANION in modifiers
-    internal val isInlineClass get() = kind == Kind.CLASS && INLINE in modifiers
+    internal val isInlineOrValClass get() = kind == Kind.CLASS &&
```

Comment:
IIRC they're planning to remove `inline` at some point, but I guess we can rename all occurrences of `inlineOrValue` to `value` then.

Reply:
Yeah I thought about that. Since it's internal we can always change later, i felt like this was more clear for now. Thoughts?

Reply:
Agree!